app/src/main/java/com/example/parkingsystemjava/MainActivity.java

```diff
@@ -1,14 +1,17 @@
 package com.example.parkingsystemjava;
-
 import androidx.appcompat.app.AppCompatActivity;
-
 import android.os.Bundle;
+import android.widget.Button;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Button buttonmain = findViewById(R.id.button_main_parking_size);
```

Comment:
as you are not using the button, please remove it

Reply:
done